ObsLocTAP.tex

```diff
@@ -554,6 +559,28 @@ \subsection{Observation Locator data model }
 removed from the planning table or they could return to the "Planned" status
 for a possible future schedule.
 
+Tracking type (\textbf{tracking\_type}) is an optional field of the obsplan table
+to characterize the tracking foreseen during the observation execution.  
+The default value would be \textbf{RA/Dec}. 
+For this observations the pointing is mantained on the same position on the sky
+during all the observation period. This kind of tracking is usually known as
+siderial tracking for ground based observations. 
+
+Other possible value is \textbf{solar-system-object-tracking}, to be used when
+the pointing of the observatory varies to maintain the selected solar system
+target on the same position of the observatory point of view. In this case, the
+values for \textbf{s\_ra} and \textbf{s\_dec} could be defined as null or,
+if possible, a representative position within the expected siderial movement.
+In this case, \textbf{target\_name} would be an identifier of the solar system
+object.
```

Comment:
`solar system object`  =>  `solar system or moving object` 

I'm thinking of artificial satellites around Earth, or debris, for instance. Or the objects like `1I/ʻOumuamua` which may not be solar system objects (or only temporarily)...

Reply:
OK. I will modify all the things and upload it again